support-frontend/assets/components/thankYou/thankYouModuleData.tsx

```diff
@@ -55,9 +66,15 @@ export const getThankYouModuleData = (
 	isOneOff: boolean,
 	amountIsAboveThreshold: boolean,
 	campaignCode?: string,
+	supportReminder?: ThankYouSupportReminderState,
+	feedbackSurveyHasBeenCompleted?: boolean,
 ): Record<ThankYouModuleType, ThankYouModuleData> => {
-	const { feedbackSurveyHasBeenCompleted, supportReminder } =
-		useContributionsSelector((state) => state.page.checkoutForm.thankYou);
+	const [feedbackSurveyCompleted, SetFeedbackSurveyCompleted] =
+		useState<boolean>(feedbackSurveyHasBeenCompleted ?? false);
+	const [supportRemind, SetSupportRemind] =
```

Comment:
```suggestion
	const [supportRemind, setSupportRemind] =
```

Small code consistency.

Reply:
Ah yes, no worries, updated